bitbots_vision/config/visionparams.yaml

```diff
@@ -40,30 +60,19 @@ blue_color_detector_upper_values_h: 116
 blue_color_detector_upper_values_s: 255
 blue_color_detector_upper_values_v: 255
 
-field_boundary_finder_search_method: 'dynamic'  # iteration, reversed, binary or dynamic
-field_boundary_finder_vertical_steps: 30
-field_boundary_finder_horizontal_steps: 30
-field_boundary_finder_roi_height: 1
-field_boundary_finder_roi_width: 1
-field_boundary_finder_roi_increase: 0.08  # increase of kernel radius per pixel
-field_boundary_finder_green_threshold: 150
-field_boundary_finder_precision_pix: 5
-field_boundary_finder_min_precision_pix: 3
-field_boundary_finder_head_joint_threshold: -0.5
+field_boundary_detector_search_method: 'reversed'  # iteration, reversed, binary or dynamic
+field_boundary_detector_vertical_steps: 30
+field_boundary_detector_horizontal_steps: 30
+field_boundary_detector_roi_height: 1
+field_boundary_detector_roi_width: 1
+field_boundary_detector_roi_increase: 0.1  # increase of kernel radius per pixel
+field_boundary_detector_green_threshold: 60
+field_boundary_detector_precision_pix: 5
+field_boundary_detector_min_precision_pix: 3
+field_boundary_detector_head_tilt_threshold: 20
 
-vision_ball_classifier: 'yolo'  # fcnn or yolo or dummy
-vision_publish_debug_image: false
-vision_publish_field_mask_image: false
-vision_parallelize: true
-vision_use_sim_color: false
-vision_ball_candidate_field_boundary_y_offset: 0
-vision_ball_candidate_rating_threshold: 0.5
-vision_debug_printer_classes: ''
-vision_blind_threshold: 30
-
-line_detector_field_boundary_offset: 15
-line_detector_linepoints_range: 0
-line_detector_blur_kernel_size: 9
```

Comment:
we should use the parameter instead of removing it.

Reply:
Seems right. But let's implement this after version 1 okay? We should open an issue. Is it okay if these 3 lines are removed from the config until then?

Reply:
After the issue is opened, I'm fine with it.